### ISSUE TITLE
[BUG] textValueMap parsing emoticons shows "No valid Y values found in note"

### ISSUE THREAD

**Issue by OmniNaut** [pyrochlore/obsidian-tracker#237]
Trying to map emojis against numbers to put in a line graph, but no matter what I try emojis as original value always shows "No valid Y values found in note". If i change the original value to text or number textValueMap works fine.
I use inline metadata. I tried to change to frontmatter but the bug still exists.
I used this example. https://github.com/pyrochlore/obsidian-tracker/blob/master/examples/TestTextValueMap.md

This is the code I used:

```
searchType: dvfield
searchTarget: Physical, Mental, Emotional, Spiritual, Creative, Peaceful
folder: Tracking/Daily
startDate: 2023-06-05
endDate: 2023-06-11
textValueMap:
	😂: 6
	🙂: 5  
	😐: 4
	☹️: 3
	😞: 2
	😭: 1
datasetName: 🚹 Physical, 🧠 Mental, 😶 Emotional, ☮️ Spiritual, 🎨 Creative, ✌🏼 Peaceful
fixedScale: 1.25
margin: 2
line:
    lineColor: yellow, orange, lightblue, red, pink, green
    lineWidth: 3
    showLegend: true
    legendPosition: right
```

Unfortunately since the developers hasn't be active for over a year, I consider this plugin abandoned. So maybe there is a work around  we as a community can come up with. At least it's a documentation of the issue.

Plugin version: 1.10.9
Obsidian: 1.4.2 (Insider)
Windows 11 22H2

**Comment by DarkRedman** [comment]
I have a similar issue at the exception that it doesn't apply the mapping and looks like a normal line chart.

**Comment by jarpolacek** [comment]
My calendar crashed when I changed type of field to boolean. Any advice?
```tracker
searchType: frontmatter
searchTarget: dřepy
folder: Deník/Dny
datasetName: Kdy jsi dělal dřepy
month:
	startWeekOn: "mon"
	color: lightblue
```
![image](https://github.com/pyrochlore/obsidian-tracker/assets/149304948/f4bb3484-e457-4062-bddb-8c3c93d1afe3)

**Comment by gnashedteeth** [comment]
Would love it if dvField could handle inline key::value pairs that utilize emojis. All of the metadata I want to visualize in my daily notes is inline and uses [🏢:: value] syntax...

**Comment by lazyguru** [comment]
#196 _might_ be what you are after @gnashedteeth?

**Comment by lazyguru** [comment]
@jarpolacek can you try with the latest version? Support for boolean types was added recently

**Comment by gnashedteeth** [comment]
> #196 _might_ be what you are after @gnashedteeth?

Could be! Investigating. Thanks for taking over this project!!

**Comment by lazyguru** [comment]
I believe I have solved the original issue reported by @OmniNaut and will add it to #196 shortly and release it

**Comment by lazyguru** [comment]
This should be fixed now with version 1.13.0
